src/remoted/request.c

```diff
@@ -54,15 +48,7 @@ void * req_main(__attribute__((unused)) void * arg) {
 
     mdebug1("Running request listener thread.");
 
-    // Get values from internal options
-
-    request_pool = getDefine_Int("remoted", "request_pool", 1, 4096);
-    request_timeout = getDefine_Int("remoted", "request_timeout", 1, 600);
-    response_timeout = getDefine_Int("remoted", "response_timeout", 1, 3600);
-    rto_sec = getDefine_Int("remoted", "request_rto_sec", 0, 60);
-    rto_msec = getDefine_Int("remoted", "request_rto_msec", 0, 999);
-    max_attempts = getDefine_Int("remoted", "max_attempts", 1, 16);
-    guess_agent_group = getDefine_Int("remoted", "guess_agent_group", 0, 1);
+    request_pool = logr.request_pool;
```

Comment:
Is this variable really necessary?

Reply:
It's necessary to keep the original value set in the option in order to show it in the on-demand configuration as this variable changes its value depending on the circumstances:
https://github.com/wazuh/wazuh/blob/86ccb6cd55511e5d9727c8a8bfcd0465720e6371/src/remoted/request.c#L370